### DISCUSSION TITLE
Final updates

### DISCUSSION THREAD

**Discussion by danielsgranja** [munoztd0/DAP_2021#21] [General]
Hello David!

So, i've been able to link my RStudio to the Git depository again, and have been progressing on the DAP, but i cannot knit the document in pdf, so i'm working with html. 

However, i have a problem with the diagnostic of the mixed-effect models. I think it's way too complex for the time I have left, so i don't know what to do. 

Besides that, i still have to look at the appendix and at the references, and finish the conclusion text. I will try to finish it for tomorrow. 

Thanks!

**Comment by munoztd0** [comment]
I don't really know what to tell you here... You had time during the semester to get things going, but right now of course things are going to be hard to finish..

Just do as much as you can and we will see where it goes.
